frontend/src/components/events/organisms/event_section.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Text from "../atoms/text/text";
+import { EventCard } from "../molecules/event_card";
+import Flex, { FlexItem } from "../atoms/flex/flex";
+import defaultLogo from "../../../assets/ymim1.png";
+
+const EventSection = ({ events, isUpcoming }) => {
+  let sectionTitle = "Upcoming Events";
+  if (isUpcoming == false) {
+    events.reverse();
+    sectionTitle = "Past Events";
+  }
+  return (
+    <div>
+      <Text text={sectionTitle} type="heading" />
+      <Flex>
+        {events.map(({ name, start, logo, url, id, status }) => {
+          if (
+            status == "completed" ||
```

Comment:
we want all these to be the triple equals `===`
https://stackoverflow.com/questions/523643/difference-between-and-in-javascript

Reply:
this logic should be handled in the index i believe -> check my comment there to see how